build/debian-hyperkube-base/Dockerfile

```diff
@@ -35,6 +35,8 @@ RUN echo CACHEBUST>/dev/null && clean-install \
     openssh-client \
     nfs-common \
     socat \
-    util-linux
+    util-linux && \
+    echo "deb http://ftp.debian.org/debian jessie-backports main contrib" > /etc/apt/sources.list.d/jesse-backports.list \
+    && clean-install zfsutils-linux
```

Comment:
We shouldn't call clean-install twice -- this involves downloading/using/deleting all the apt-caches twice.

Reply:
Oh, okay. I didn't want to interfere with any package resolution if there was a hard dependency on something in Jessie, not in Jessie backports.